### ISSUE TITLE
Windows error Failed to load dynamic library

### ISSUE THREAD

**Issue by DamienMrtl** [gonuit/rps#20]
Hi, i'm getting this error on windows 11 with flutter.

```
Flutter 3.19.5 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 300451adae (2 weeks ago) • 2024-03-27 21:54:07 -0500
Engine • revision e76c956498
Tools • Dart 3.3.3 • DevTools 2.31.1
```

<img width="1307" alt="image" src="https://github.com/gonuit/rps/assets/7550391/2a48deef-dc1c-4b67-9f9d-19366ce48cca">

**Comment by gwijaya94** [comment]
same here, I tried with script `rps hello` and it throws the same error.
Hmm is it because of windows 11 ? haven't tried it on win10.

**Comment by notesz5** [comment]
Same issue here on Win11. It works just fine on Win10.

**Comment by ebarcsa** [comment]
Same issue on recently installed win10, after running with `dart pub global activate rps --version 0.8.1` and trying to run a script with rps. I've added `%LOCALAPPDATA%\Pub\Cache\bin` to path manually.
Intrestingly, if I navigate to `%LOCALAPPDATA%\Pub\Cache\hosted\pub.dev\rps-0.8.1\native` then I can see the `rps_x64.dll`.
Maybe some dependencies of this dll cannot be resolved.

**Comment by ptvinhtech13** [comment]
I got the same issue. 
![image](https://github.com/user-attachments/assets/f1afb6f0-af50-4295-9c0d-8c4ef40c4ee5)
ps: macos works well with this

**Comment by alanmelone** [comment]
I got the same issue on Win 11. The solution is simple. Just install Visual Studio with recommended `Desktop development with C++` option from recommendation of `flutter doctor` command.

But it would be very nice to make work this package without installing Visual Studio.

**Comment by gonuit** [comment]
This is probably due to the lack of CRT dynamic lib. I will look into this.

**Comment by gonuit** [comment]
Hi all,
A possible fix for this bug has been implemented and is present in [rps version 0.9.1](https://pub.dev/packages/rps/versions).

I am closing this issue for now, if this error still occurs, feel free to reopen this issue.
